### PR TITLE
Create sbt plugin

### PR DESCRIPTION
**Pull request by Giovannini**

### CODE REVIEW

**Comment by Giovannini** [comment]
test this please

**Comment by Giovannini** [comment]
test this please

**Comment by Giovannini** [comment]
test this please

File: src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ class EvenFibonnacciTest extends WordSpec {
   "fibonnacci" should {
```
Comment (khand19) [inline src/project/src/test/scala/io/giovannini/EvenFibonnacciTest.scala:7]: List(1) did not equal List(3)

**Pull request merged**